pvlib/tests/test_irradiance.py

```diff
@@ -612,7 +612,7 @@ def test_disc_value(pressure, expected):
     # see GH 449 for pressure=None vs. 101325.
     columns = ['dni', 'kt', 'airmass']
     times = pd.DatetimeIndex(['2014-06-24T1200', '2014-06-24T1800'],
-                             tz='America/Phoenix')
+                             tz='utc').tz_convert('America/Phoenix')
```

Comment:
This had prevented `test_disc_value` from failing. I reverted `times` to the original but dealt with it like `test_get_extra_radiation_nrel_numba` because the previous answer to the localized timezone was the same with the one with its attribute replaced nothing else but its tz with the 'UTC'.